adapters/triplelift/triplelift.go

```diff
@@ -26,7 +26,7 @@ type TripleliftRespExt struct {
 
 func getBidType(ext TripleliftRespExt) openrtb_ext.BidType {
 	t := ext.Triplelift.Format
-	if t == 11 {
+	if t == 11 || t == 12 || t == 17 {
```

Comment:
Would it be better to extract these magic numbers to constants?

Reply:
Ideally, I agree. I do not know what these format values represent. I don't think it matters too much in this context and I'm personally ok with magic numbers in this instance. Do you feel strongly about this suggestion or is this ok?

Reply:
These are just different ids indicating instream, outstream, and one other video format. I don't think it's super relevant to have that indicated here.